codegen/specs/stytch/sessions.yml

```
@@ -25,6 +25,9 @@ methods:
         arg_type: Optional[Dict[str, Any]] = None
     response_type:
       session: StytchSession
+      session_jwt: str
+      session_token: str
+      user: Optional[User]
```

From our response type definition, this field is always populated. If you've seen it be missing, I'd say there's a bug somewhere!
```suggestion
      user: User
```

---

This is unfortunately because we support `authenticate_jwt_local` which -- as far as I can tell -- *cannot* get the `User` object.

https://github.com/stytchauth/stytch-python/blob/main/stytch/api/sessions.py#L180

If I'm wrong about that, that would be awesome, though!

---

Ah, right, that's true. I checked `stytch-node` for comparison, and `authenticateJwtLocal` there has a totally different return value that's just the session: https://github.com/stytchauth/stytch-node/blob/8594b938dbf44902adeb6e7e29b810d7cee3c305/lib/sessions.ts#L196-L203

In that case, I think we have to either make this optional or split the return types to narrow the `authenticate_jwt_local` one.

I feel like different return types is the "right" way because `stytch-go` does the same thing: https://github.com/stytchauth/stytch-go/blob/82dbfef28e13695c7ea4b4d262399c8d0fbf97b2/stytch/session/session.go#L84-L87

---

Let me think about it -- strictly speaking, changing that return type would be a major update, which sounds really gross, though. If we _just_ returned the session, it would be reasonable, but we'd also probably want to entirely get rid of the `authenticate_jwt` method unless we wanted to strip everything out of the session.

Alternatively, we could have `authenticate_jwt` return a `Union[AuthenticateResponse, LocalAuthenticateResult]`

---

Those all make sense to me. If we don't want to do a major, I think it's fine to make the field optional with a TODO(v7) comment to split the return types back into what they really are.

Can we add something to the `authenticate_jwt_local` docstring that mentions that the field is always `None` there?

---

Yeah, I'll do that. I was thinking about it and while I'm not opposed to making the change, I don't want to cause thrash by going to v7 so quickly after we just announced v6 in the changelog yesterday haha

---

Nice, ship it! 🚀